src/main/java/org/sefglobal/scholarx/model/BulkEmailDto.java

```diff
@@ -0,0 +1,46 @@
+package org.sefglobal.scholarx.model;
+
+import lombok.Data;
+import org.sefglobal.scholarx.util.MailGroup;
+
+import java.util.List;
+
+@Data
+public class BulkEmailDto {
+	private String subject;
+	private String message;
+	private MailGroup mailGroup;
+	private List<String> additionalEmails;
+
+	public String getSubject() {
+		return subject;
+	}
```

Comment:
We don't need to write getters and setters because you have imported lamblok

Reply:
Removed lombok